README.md

```diff
@@ -109,14 +101,6 @@ Note that for all merges from master or stable, the merge should pick up:
 But never:
 * changes that bumped to a SHA inside config/components
 
-Here's a sample snippet used for a stable -> aardwolf merge:
```

Comment:
Yeah, good call. This could be used as the template for stable->master merges (replacing aardwolf with master throughout of course), since those merges will have the same exceptions for changes to various items in `configs/components`.